README.md

```diff
@@ -290,11 +290,11 @@ Once validated in `development`, changes can be promoted to `non-production` by
 
 ### Terraform-validator
 
-This repo uses [terraform-validator](https://github.com/GoogleCloudPlatform/terraform-validator) to validate the terraform plans against Forseti Security Config Validator [Policy Library](https://github.com/forseti-security/policy-library).
+This repo uses [terraform-validator](https://github.com/GoogleCloudPlatform/terraform-validator) to validate the terraform plans against a Constraint Framework [Policy Library](https://github.com/GoogleCloudPlatform/policy-library).
```

Comment:
I'm not sure we want to talk about "Constraint Framework" too much publicly. It's not a branded term.

```suggestion
This repo uses [terraform-validator](https://github.com/GoogleCloudPlatform/terraform-validator) to validate the terraform plans against a [library of GCP policies](https://github.com/GoogleCloudPlatform/policy-library).
```